nextjs-project/app/globals.css

```
@@ -26,6 +26,12 @@
     0 0 10px 0 rgba(255, 255, 255, 0.05) inset;
 }
 
+/* Animations — inherited from The Monolith reference */
+@keyframes slideUp {
```

Stylelint (17.9.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)